BaseTools/Source/C/GenFfs/GenFfs.c

```diff
@@ -7,7 +7,9 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 **/
 
 #ifndef __GNUC__
+#define RUNTIME_FUNCTION  _WINNT_DUP_RUNTIME_FUNCTION

```

Comment:
This is all trying to not edit the RUNTIME_FUNCTION inside PeImage header by following the EmulatorPkg style. Not sure if this is really worth it. Otherwise, we can fallback to the style here: https://github.com/microsoft/mu_basecore/pull/934/commits/7fdfb938b39cd30163d42bed2ea3f62d9957b72a#diff-f77dc7ec6832c450b9e4d67f703d941614a34505c867973366ca8acc15f056b4

Reply:
I think this way is OK.